File: src/test/java/test/RevokeRoleTest.java
import cn.authing.sdk.java.client.ManagementClient;
import cn.authing.sdk.java.dto.*;
import cn.authing.sdk.java.model.ManagementClientOptions;
import cn.authing.sdk.java.util.JsonUtils;

import java.util.Collections;



public class RevokeRoleTest {

    private static final String ACCESS_KEY_ID = "YOUR_ACCESS_KEY_ID";
    private static final String ACCESS_KEY_SECRET = "YOUR_ACCESS_KEY_SECRET";


    public static void main(String[] args) throws Throwable {
        ManagementClientOptions clientOptions = new ManagementClientOptions(ACCESS_KEY_ID, ACCESS_KEY_SECRET);
        ManagementClient client = new ManagementClient(clientOptions);

        RevokeRoleDto request = new RevokeRoleDto();
        request.setTargets(Collections.singletonList(new cn.authing.sdk.java.dto.TargetDto()));
        request.setCode("code_9419");
        request.setNamespace("namespace_2785");

        IsSuccessRespDto response = client.revokeRole(request);
        System.out.println(JsonUtils.serialize(response));
    }

}

File: src/test/go/test/RevokeRoleTest.go
package main

import (
"authing-go-sdk/client"
"authing-go-sdk/dto"
"fmt"
)

func main() {
    options := client.ManagementClientOptions{
        AccessKeyId:     "AUTHING_USERPOOL_ID",
        AccessKeySecret: "AUTHING_USERPOOL_SECRET",
    }
    var err error
    client, err := client.NewClient(&options)
    if err != nil {
        panic(err)
    }
    request := dto.RevokeRoleDto{
        Targets: nil,
        Code: "code_5272",
        Namespace: "namespace_3830",
    }
    response := client.RevokeRole(&request)
    fmt.Println(response)
}

